pages/auth/user-profile/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { NextPage } from 'next';
+import React from 'react';
+import { featureFlags } from '../../../feature-flags/feature-flags';
+import { CURRENT_ENVIRONMENT } from '../../../src/config/endpoints';
+import { useAuthContext } from '../../../src/core/context/AuthContext';
+import UserProfile from '../../../src/stories/containers/users/user-profile/user-profile';
+import NotFoundPage from '../../404';
+
+const UserProfilePage: NextPage = () => {
+  const { authToken } = useAuthContext();
+
+  if (!featureFlags[CURRENT_ENVIRONMENT].FEATURE_AUTH || !authToken) {
```

Comment:
this should be handled in SSR using `getServerSideProps` instead returning not found in order to let nextjs handle the 404 error

Reply:
In fact it was my mistake , its not a page, so I will remove from the page directory .